src/ucp/tag/rndv.c

```diff
@@ -226,6 +227,47 @@ ucs_status_t ucp_tag_send_start_rndv(ucp_request_t *sreq)
     return status;
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_rndv_adjust_zcopy_length(size_t min_zcopy, size_t max_zcopy, size_t align,
```

Comment:
maybe better return resulted `length` instead of making it in/out argument

Reply:
done